src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -9,8 +9,20 @@ extern s16 D_8082AAEC[];
 extern s16 D_8082AB2C[];
 extern u8 gSlotAgeReqs[];
 extern u8 gEquipAgeReqs[EQUIP_TYPE_MAX][4];
+extern u8 gItemAgeReqs[];
 extern u8 gAreaGsFlags[];
 
+#define AGE_REQ_ADULT LINK_AGE_ADULT
+#define AGE_REQ_CHILD LINK_AGE_CHILD
+#define AGE_REQ_NONE 9
+
+#define CHECK_AGE_REQ_SLOT(slot) \
+    ((gSlotAgeReqs[slot] == AGE_REQ_NONE) || gSlotAgeReqs[slot] == ((void)0, gSaveContext.save.linkAge))
+#define CHECK_AGE_REQ_EQUIP(y, x) \
+    ((gEquipAgeReqs[y][x] == AGE_REQ_NONE) || (gEquipAgeReqs[y][x] == ((void)0, gSaveContext.save.linkAge)))
+#define CHECK_AGE_REQ_ITEM(item) \
+    ((gItemAgeReqs[item] == AGE_REQ_NONE) || (gItemAgeReqs[item] == ((void)0, gSaveContext.save.linkAge)))
```

Comment:
id line break between each of these but not a huge deal

Reply:
I'm conflicted because on one hand sure
on the other hand there's a lot of stuff in this header in the full draft (and too many line breaks weaken them as separators) https://github.com/Dragorn421/oot/blob/d43613694caf4574f2375c22529de9918caee054/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

Reply:
if you need to separate things by subsystem or something, comments on their own line would suffice (only if really needed). I see this as more of a readability issue than separators. I cant read a macro very well if its all smushed up with other ones surrounding it